pkg/alertmanager/api.go

```diff
@@ -36,7 +36,7 @@ type UserConfig struct {
 func (am *MultitenantAlertmanager) GetUserConfig(w http.ResponseWriter, r *http.Request) {
 	logger := util.WithContext(r.Context(), am.logger)
 
-	userID, err := user.ExtractOrgID(r.Context())
+	userID, err := tenant.DefaultResolver().TenantID(r.Context())
```

Comment:
**suggestion(if-minor):** Instead of retrieving the default resolver and calling these functions, what if you expose the functions of the `Resolver` interface as top level functions in the `tenant` package and just assume they will utilize the default in the package. That could make the syntax a bit cleaner.

```suggestion
	userID, err := tenant.TenantID(r.Context())
```

Reply:
I have done that, golint complained about the stuttering issue: `tenant.TenantID[s]`, but I think it's important to have it there to make it clear UserID vs TenantID.